ui/components/connections/index.js

```diff
@@ -403,6 +481,21 @@ function Connections({ classes, updateProgress }) {
     notify({ message : `${action.error_msg}: ${error}`, event_type : EVENT_TYPES.ERROR, details : error.toString() })
   };
 
+  const handleStatusChange = (e, connectionId) => {
+    const requestBody = JSON.stringify({
+      "status" : e.target.value,
+    });
```

Comment:
Hi, @senali-d. 
**`Put` request needs you to send the entire details of the connection, not only status.**

You can do the same (what you did) using a `patch` request, but the `patch` method is unavailable for the endpoint.
Can you please modify this and let us know the result?

Reply:
@Aritra8438 Ok. I'll check this. But all the other fields are not null except name in this way (using `put` method).

Reply:
Alternatively, can you check that sending `name` along with `status` fixes this error or not?

Reply:
Any other field didn't give null except 'name'. Actually not `null`. `name` filed is not return after update.

Reply:
> Alternatively, can you check that sending `name` along with `status` fixes this error or not?

Can you check this? This might be a fix.

Reply:
> Any other field didn't give null except 'name'. Actually not `null`. `name` filed is not return after update.

Oh! That's even weirder.